packages/api/methods.js

```diff
@@ -149,19 +149,3 @@ export function updatePayee(id, fields) {
 export function deletePayee(id) {
   return send('api/payee-delete', { id });
 }
-
-export function getPayeeRules(payeeId) {
-  return send('api/payee-rules-get', { payeeId });
-}
-
-export function createPayeeRule(payeeId, rule) {
-  return send('api/payee-rule-create', { payee_id: payeeId, rule });
-}
-
-export function updatePayeeRule(id, fields) {
```

Comment:
There's a few more of these in `packages/loot-core/src/server/db/index.ts`

Reply:
I don’t see any?

Reply:
https://github.com/actualbudget/actual/blob/master/packages/loot-core/src/server/db/index.ts#L545

Reply:
Honestly.. I am blind. You already removed these.